src/js/constants/Networking.js

```diff
@@ -6,7 +6,8 @@ const Networking = {
     USER: 'USER'
   },
   L4LB_ADDRESS: '.marathon.l4lb.thisdcos.directory',
-  VIP_LABEL_REGEX: /^VIP_[0-9]+$/
+  VIP_LABEL_REGEX: /^VIP_[0-9]+$/,
+  VIP_LABEL_VALUE_REGEX: /.+:\d+/
```

Comment:
Please change the  pattern to`([\w\d\_\-\.\/]+):(\d+)` to avoid false positives.